html/pfappserver/root/static.alt/src/views/Configuration/_components/SwitchesImport.vue

```diff
@@ -1,198 +1,105 @@
-    },
-    escapeKey () {
-      return this.$store.getters['events/escapeKey']
+      files: [],
+      tabIndex: 0,
+      defaultStaticMapping: [],
+      importFields, // ../_config/switch
+      isLoading: false
     }
   },
   methods: {
-    init () {
-      this.$store.dispatch('$_roles/all').then(data => {
-        this.roles = data
-      })
-      if (this.id) {
-        // existing
-        this.$store.dispatch(`${this.storeName}/optionsById`, this.id).then(options => {
-          this.options = options
-          this.$store.dispatch(`${this.storeName}/getSwitch`, this.id).then(form => {
-            if (this.isClone) form.id = `${form.id}-${this.$i18n.t('copy')}`
-            this.form = form
-            this.switchGroup = form.group
-          })
+    abortFile (index) {
+      this.files[index].reader.abort()
+    },
+    closeFile (index) {
+      const file = this.files[index]
+      file.close()
+    },
+    importPromise (payload, dryRun) {
+      return new Promise((resolve, reject) => {
+        this.$store.dispatch('$_switches/bulkImport', payload).then(result => {
+          // do something with the result, then Promise.resolve to continue processing
+          resolve(result)
+        }).catch(err => {
+          // do something with the error, then Promise.reject to stop processing
+          reject(err)
         })
-      } else {
-        // new
-        this.$store.dispatch(`${this.storeName}/optionsBySwitchGroup`, this.switchGroup).then(options => {
-          this.options = options
-          this.form = defaults(options.meta) // set defaults
-          this.form.group = this.switchGroup
-        })
-      }
+      })
     },
     close () {
       this.$router.push({ name: 'switches' })
-    },
-    clone () {
-      this.$router.push({ name: 'cloneSwitch' })
-    },
-    create () {
-      const actionKey = this.actionKey
-      this.$store.dispatch(`${this.storeName}/createSwitch`, this.form).then(response => {
-        if (actionKey) { // [CTRL] key pressed
-          this.close()
-        } else {
-          this.$router.push({ name: 'switch', params: { id: this.form.id } })
-        }
-      })
-    },
-    save () {
-      const actionKey = this.actionKey
-      this.$store.dispatch(`${this.storeName}/updateSwitch`, this.form).then(response => {
-        if (actionKey) { // [CTRL] key pressed
-          this.close()
-        }
-      })
-    },
-    remove () {
-      this.$store.dispatch(`${this.storeName}/deleteSwitch`, this.id).then(response => {
-        this.close()
-      })
-    }
-  },
-  created () {
-    this.init()
-  },
-  watch: {
-    id: {
-      handler: function (a, b) {
-        this.init()
-      }
-    },
-    isClone: {
-      handler: function (a, b) {
-        this.init()
-      }
-    },
-    escapeKey (pressed) {
-      if (pressed) this.close()
     }
   }
 }
 </script>
+
+<style lang="scss">
+.nav-tabs > li > a,
+.nav-pills > li > a {
+  margin-right: 0.5rem!important;
```

Comment:
This will affect all `b-nav` instances, so make sure it's required and move it to `src/styles/_nav.scss`.